modules/library/referencing/src/main/java/org/geotools/measure/Units.java

```diff
@@ -115,11 +117,16 @@ private Units() {}
 
     public static final Unit<Time> YEAR = SI.YEAR;
 
-    static final UnitFormat format = SimpleUnitFormat.getInstance();
+    static final UnitFormat format = GeoToolsUnitFormat.getInstance();
 
     static {
         /** Associates the labels to units created in this class. */
         registerCustomUnits((SimpleUnitFormat) format);
+        // not accessible, causes testGTDefinedFormats to fail:
+        // registerCustomUnits(GeoToolsCRSUnitFormat.getInstance(Citations.EPSG);
+        // registerCustomUnits(GeoToolsCRSUnitFormat.getInstance(Citations.ESRI);
+        // ideally, we would call `registerCustomUnits` in the `BaseGT2Format` constructor,
+        // but that would introduce a dependency cycle between gt-metadata and gt-referencing
```

Comment:
@aaime what to do here?

Reply:
I have no good suggestion... don't have enough review time to wrap my head around the changes happening here (I have 4 hours total a week for reviews that are not related to active contracts, that has to be shared among all pending pull requests).